profitbricks-rest/src/main/java/org/apache/jclouds/profitbricks/rest/ProfitBricksProviderMetadata.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.jclouds.profitbricks.rest;
+
+import com.google.auto.service.AutoService;
+import java.net.URI;
+import java.util.Properties;
+import static org.jclouds.location.reference.LocationConstants.ISO3166_CODES;
+import static org.jclouds.location.reference.LocationConstants.PROPERTY_REGION;
+import static org.jclouds.location.reference.LocationConstants.PROPERTY_REGIONS;
+import org.jclouds.providers.ProviderMetadata;
+import org.jclouds.providers.internal.BaseProviderMetadata;
+
+@AutoService(ProviderMetadata.class)
+public class ProfitBricksProviderMetadata extends BaseProviderMetadata {
+
+   public static Builder builder() {
+      return new Builder();
+   }
+
+   @Override
+   public Builder toBuilder() {
+      return builder().fromProviderMetadata(this);
+   }
+
+   public ProfitBricksProviderMetadata() {
+      super(builder());
+   }
+
+   public ProfitBricksProviderMetadata(Builder builder) {
+      super(builder);
+   }
+
+   public static Properties defaultProperties() {
+      Properties properties = ProfitBricksApiMetadata.defaultProperties();
+      properties.setProperty(PROPERTY_REGIONS, "DE,US");
+      properties.setProperty(PROPERTY_REGION + ".DE.FKB." + ISO3166_CODES, "DE-FKB");
+      properties.setProperty(PROPERTY_REGION + ".DE.FRA." + ISO3166_CODES, "DE-FRA");
+      properties.setProperty(PROPERTY_REGION + ".US.LAS." + ISO3166_CODES, "US-LAS");
+      properties.setProperty(PROPERTY_REGION + ".US.LASDEV." + ISO3166_CODES, "US-LASDEV");
```

Comment:
It looks like the provider has zones and regions. I assume the region IDs are "DE" and "US", and the zone IDs are "FKB, "FRA", "LAS". The right way to configure this in jclouds is as follows:

```java
properties.setProperty(PROPERTY_ISO3166_CODES, "US-NV,DE-HE,DE-BW");

properties.setProperty(PROPERTY_REGIONS, "DE,US");
properties.setProperty(PROPERTY_REGION + ".DE.zones", "FKB,FRA")
properties.setProperty(PROPERTY_REGION + ".US.zones", "LAS");

properties.setProperty(PROPERTY_ZONES, "FKB,FRA,LAS");
properties.setProperty(PROPERTY_ZONE + ".FKB." + ISO3166_CODES, "DE-BW");
properties.setProperty(PROPERTY_ZONE + ".FRA." + ISO3166_CODES, "DE-HE");
properties.setProperty(PROPERTY_ZONE + ".LAS." + ISO3166_CODES, "US-NV");
```

Note that I've already fixed the ISO codes to use the right ones. Also, the zone IDs and regions IDs used here are up to you. Feel free to change them to something more user-friendly, as those "FKB/FRA/LAS" is what users will use as the "locationId" when choosing where they want to deploy the VMs.